Source/Scene/GltfVertexBufferLoader.js

```diff
@@ -178,13 +195,22 @@ function getQuantizationInformation(dracoQuantization, componentDatatype) {
   quantization.octEncoded = dracoQuantization.octEncoded;
   quantization.normalizationRange = range;
 
-  if (!quantization.octEncoded) {
-    quantization.quantizedVolumeOffset = Cartesian3.unpack(
-      dracoQuantization.minValues
-    );
-    quantization.quantizedVolumeDimensions = new Cartesian3(
-      dracoQuantization.range
-    );
+  if (quantization.octEncoded) {
+    quantization.type = AttributeType.VEC2;
+  } else {
+    var MathType = AttributeType.getMathType(type);
+    if (MathType === Number) {
+      quantization.quantizedVolumeOffset = dracoQuantization.minValues[0];
+      quantization.quantizedVolumeDimensions = dracoQuantization.range;
+    } else {
+      quantization.quantizedVolumeOffset = MathType.unpack(
+        dracoQuantization.minValues
+      );
+      quantization.quantizedVolumeDimensions = MathType.unpack(
+        new Array(componentCount).fill(dracoQuantization.range)
```

Comment:
`Array.fill()` is [not supported in IE](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/fill#browser_compatibility)

Reply:
:facepalm: